packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -22,6 +22,12 @@ import 'ticker_provider.dart';
 /// "routes" that are pushed on and popped off the navigator. Most routes have
 /// visual affordances, which they place in the navigators [Overlay] using one
 /// or more [OverlayEntry] objects.
+///
+/// See [Navigator] for more explanation of how to use a Route
+/// with navigation, including code examples.
+///
+/// See [material.MaterialPageRoute] for a route that replaces the
```

Comment:
It will but you don't need to do that. It'll find MaterialPageRoute regardless. House style is to not be explicit like that.